srcpkgs/caddy/files/caddy/run

```diff
@@ -2,7 +2,7 @@
 
 [ -r ./conf ] && . ./conf
 
-export CADDYPATH=/var/lib/caddy
+export HOME=/var/lib/caddy
 ulimit -n ${MAX_OPEN_FILES:-8192}
-cd /etc/caddy
-exec chpst -u caddy caddy
+
+exec chpst -u caddy caddy run --environ --config /etc/caddy/Caddyfile
```

Comment:
This file doesn't exist in the package... Does it still work regardless?

Reply:
Good catch!

I've added an empty `Caddyfile`

Reply:
Don't forget to add this to `conf_files`.

Reply:
Thanks for pointing out!

I've added it.

Reply:
Maybe 

```suggestion
exec chpst -u caddy caddy run --environ --config ${CONFFILE:-/etc/caddy/Caddyfile}
```

How does `caddy` behave if you don't give it a config file? Maybe we shouldn't start requiring one...